api/src/scripts/translateResourceDescriptions.js

```diff
@@ -0,0 +1,126 @@
+
+/**
+ * Step 2: Call the Google API to translate those descriptions to the target language
+ *
+ * Essentially we want to feed in our array from step 1 into the google API, and save an array of objects with fields _id and translated text
+ *
+ * https://cloud.google.com/translate/docs/reference/rest/v2/translate - this is the POST request we want
+ * We can make the POST request using `fetch` - you can look in client/src/utils/auth.js for examples
+ * Make sure to specify the `source` as "en"
+ * https://stackoverflow.com/questions/35139572/sample-code-for-google-translate-api - this may help on how to format the request URL
+ *
+ * For now, you can go ahead and write this code, but I'll supply the API key later (basically, I don't want to waste API credits) - to test Step 3,
+ * you can mock out the resultant data to feed into Step 3
+ */
+const apiKey = '';
+// map ids of resources to translated descriptions
+var translatedResourceDescriptions = new Map();
+const languageType = process.argv.slice(2)[0]; // the language specified in the argument when running the script - the "target" argument in the POST request
+resourceDescriptions.forEach(function (resource) {
+  var id = resource._id;
+  var descriptionText = resource.description;
+  var source =
+    `https://www.googleapis.com/language/translate/v2?key=${apiKey}&source=en&target=${language}&callback=translateText&q=` +
+    descriptionText;
+  async (body) => {
+    const { resourceDescriptions } = body;
+    const res = await fetch(source, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        resourceDescriptions,
+      }),
+    });
+    return translatedResourceDescriptions.set(
+      id,
+      res.json().body.data.translations[0].translatedText, // store translated descriptions with id
+    );
+  };
+});
+
+/**
+ * Step 3: Store those translations back into our database
+ *
+ * You can look at our model for a translation - essentially, a "Translation" has a field "language" and then a field "messages" which is a Map of strings to other strings
+ * This is the "message ID" to its translation, so for example, if a resource had _id "1234", and translation "hola" for its description, after entering that into our db,
+ * the resulting db entry might look like:
+ * {
+ *  language: "es",
+ *  messages: {
+ *      "resource-description-6789": "gracias",
+ *      "resource-description-1234": "hola"
+ *  }
+ * }
+ *
+ * We want to loop over our results from Step 2, and enter all of those into our database in the format above ^
+ */
+
+var messagesMap = new Map();
+// store all descriptions with ids in messages map
+Object.keys(translatedResourceDescriptions).forEach(function (id) {
+  var messageKey = 'resource-description-' + id;
+  var messageValue = translatedResourceDescriptions.get(id);
+  messages.set(messageKey, messageValue);
+});
```

Comment:
One thing we can do to speed things up (and leave less room for error) - in the above step where you set your `translatedResourceDescriptions`, instead of `translatedResourceDescriptions.set(id, ...)`, I believe we can just do ```translatedResourceDescriptions.set(`resource-description-${id}`, ...)```.  Then we hopefully won't need this extra step!